src/style.css

```
@@ -0,0 +1,121 @@
+body,
+h1,
+p,
+img {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 200px;
+  max-height: 404px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
```

```suggestion
  max-width: 200px;
  border: 1px solid #f3f3f3;
  border-radius: 5px;
```

Why you need flex? The default blocks are one below the other. Add card pading here top - 32, and other 16px